pkg/crc/cluster/pullsecret.go

```diff
@@ -0,0 +1,153 @@
+package cluster
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+
+	cmdConfig "github.com/code-ready/crc/cmd/crc/cmd/config"
+	crcConfig "github.com/code-ready/crc/pkg/crc/config"
+	"github.com/code-ready/crc/pkg/crc/constants"
+	"github.com/code-ready/crc/pkg/crc/logging"
+	"github.com/code-ready/crc/pkg/crc/validation"
+	crcversion "github.com/code-ready/crc/pkg/crc/version"
+	"github.com/zalando/go-keyring"
+	terminal "golang.org/x/term"
+	"gopkg.in/AlecAivazis/survey.v1"
+)
+
+const (
+	keyringService = "crc"
+	keyringUser    = "pull-secret"
+)
+
+type PullSecretLoader interface {
```

Comment:
Why is this called `Loader` ?

Reply:
because it loads the pull secret from a store. It can load it from the keyring, a file, etc.

Reply:
for the interface that would make sense... but in the implementation I do not see this distinction,